wgpu-core/src/indirect_validation.rs

```diff
@@ -150,7 +155,7 @@ impl IndirectValidation {
                 ty: wgt::BindingType::Buffer {
                     ty: wgt::BufferBindingType::Storage { read_only: true },
                     has_dynamic_offset: true,
-                    min_binding_size: Some(std::num::NonZeroU64::new(4 * 3).unwrap()),
+                    min_binding_size: Some(NonZeroU64::new(4 * 3).unwrap()),
```

Comment:
suggestion: It's jarring to see `DST_BUFFER_SIZE` so nicely modeled above, and then see this constant. I think we should give this constant—despite only being used in this one place—so that it's easy to keep track of input and `src` and `dst` buffer sizes in the same place, which is what I'd expect for this code. Concretely, I'd offer the name `SRC_BUFFER_SIZE`.

Reply:
I'm open to having a `SRC_BUFFER_SIZE`, `DST_BUFFER_SIZE` is actually `2 * SRC_BUFFER_SIZE`.

Reply:
Pushed a `fixup!` with e7ceae48; whaddaya think?